site/contents/write_ups/2021/03-11-21_writeup.md

```diff
@@ -0,0 +1,109 @@
+- [Policy initiatives for Italy](https://oecd.ai/en/dashboards/policy-initiatives?conceptUris=http:%2F%2Fkim.oecd.org%2FTaxonomy%2FGeographicalAreas%23Italy).
+- [Italian AI white paper 2018](https://ia.italia.it/assets/whitepaper.pdf).
+
+In 2018 Italy adopted a national strategy on AI, following EU guidelines and the example of other European countries. The Italian government will release a new plan in 2022.
+
+- [Strategia nazionale per l'intelligenza artificiale - Contesto](https://www.mise.gov.it/index.php/it/strategia-intelligenza-artificiale/contesto) (in Italian).
+- [Draghi’s strategy for Italy’s AI - decode39](https://decode39.com/2162/italy-ai-strategy-draghi/).
+
+"Operating within the framework of the coordinated plan on European AI published in December 2018, Mr Draghi’s government strategy aims to increase public funds for AI research".
+
+## The report states that they want to build the “most trusted and pro-innovation system for AI governance in the world” - can both these things be true?
+
+Ideally you want to find as many bugs as you can. You cannot find all of them, as you have limited time. So at what point do you start just calling them features? What sort of prescedent does that set? Could you balance reducing the bureacracy and innovative drive? Probably not consistently. But if you consider the pharamceutical industry's approach to developing various COVID 19 vaccines, private companies were able to maintain the strict standards of their respective ethical bodies but with a streamlined process (in their case, producing batches for the next stage of the trial before they had moved along to that trial, risking wasting that entire batch should it fail the stage of the trial).
+
+We recalled studies where different people were given a software task. They found that diverse teams reduce errors because people with different backgrounds made different errors. 
+
+You will not have innovation in the long term if people do not trust your product - they will just pull the plug. A lot of the AI being commerical ties into what public thinks about it. If people don't like a product, they won't use it. Public opinion almost forms an accidental ethics board in a more consumerist, capitalist society. 
+
+Consumerism should not be the only line of defense to unethical practise. We need formal ethical boards to block unethical developments as well as guide innovators. Scientists should innovate in tramlines rather than in an open space.
+
+Facebook appear to have reached a point of innovating too hard and not building in enough data protections where we need. In order for people to trust that you're going to innovate things, you need to show you're going to say no sometimes and that there is a line in the sand. Journals seem to not have as many moral standards and are more focussed on "quality" of science and innovation. They also need to do better.
```

Comment:
```suggestion
A recent example is Facebook/Meta. Facebook appear to have reached a point of innovating too hard and not building in enough data protections, which has led to a loss of public trust. However, many of these revelations have only become public knowledge because of whistleblowers. We reflected that in order for people to trust that you can innovate responsibly, you need to show you're going to say no to new innovations sometimes and that there is a line in the sand. Similarly, Google has also lost public trust after [they fired key management figures in their Ethical AI team](https://www.bbc.co.uk/news/technology-56135817) who were trying to share research about the downsides of AI innovation. Journals are another area that seem to not have as many moral standards as we would hope and are more focussed on "quality" of science and innovation. They also need to do better at turning down papers in ML and AI without adequate ethical oversight.
```